src/hdRunner/main.cpp

```diff
@@ -275,6 +275,32 @@ class RenderSurfaceController : public oka::ResizeHandler
     }
 };
 
+void setDefaultCamera(UsdGeomCamera& cam)
+{
+    std::vector<float> r0 = {0.1269921064376831, -0.9918715357780457, -0.007994583807885647, 0};
```

Comment:
std::array

Reply:
GfMatrix4d constructor gets four std::vector<float>, why should I use array instead?